src/systemdhook.c

```diff
@@ -274,6 +274,75 @@ static bool contains_mount(const char **config_mounts, unsigned len, const char
 	return false;
 }
 
+#define RUN_PREFIX "/run/"
+
+/*
+ * Move the mounts under /run to /runtmp in the container.
+ * Finally move /runtmp tree to /run.
+ */
+static int adjust_run(const char *rootfs,
+		 const char **config_mounts,
+		 unsigned len)
+{
+	int rc = -1;
+	for (unsigned i = 0; i < len; i++) {
+		/* Skip on exact match with /run or /run/ */
+		if (!strcmp("/run", config_mounts[i]) || !strcmp("/run/", config_mounts[i])) {
+			continue;
```

Comment:
Shouldn't we use RUN_PREVIX for /run/?
Shouldn't we just exit if the user mounted over /run?
If the user does the following:
```
docker run -v /local/run:/run fedora 
```
We should not mount tmpfs on /run.